server/queue/queue-jms/src/main/java/org/apache/james/queue/jms/JMSMailQueue.java

```diff
@@ -449,10 +449,7 @@ protected void populateMail(Message message, MailImpl mail) throws JMSException
      * @return convertedValue
      */
     protected Object convertAttributeValue(Object value) {
```

Comment:
@Override ?

Reply:
Nope, the method is not exposed in the *interface*, only this class defines it to allow super class to override it (what is not done here)

Reply:
As-is, it's hardly understandable to have a method doing nothing without any comment about its existence, don't you think ?